inspirehep/modules/search/query.py

```diff
@@ -42,9 +42,12 @@ def inspire_search_factory(self, search):
     from invenio_records_rest.sorter import default_sorter_factory
 
     query_string = request.values.get('q', '')
+    # FIXME: what is the proper way to retrieve the collection?
+    # FIXME: what if someone put an invalid cc?
+    index = 'records-{}'.format(request.values.get('cc', 'hep'))
```

Comment:
@jalavik @jacquerie @jmartinm: any suggestion on how to best retrieve the index information in this context?